kroxylicious-kms-provider-hashicorp-vault-test-support/src/main/java/io/kroxylicious/kms/provider/hashicorp/vault/VaultTestKmsFacade.java

```diff
@@ -36,7 +36,7 @@
 import static java.nio.charset.StandardCharsets.UTF_8;
 
 public class VaultTestKmsFacade extends AbstractVaultTestKmsFacade {
-    private static final String HASHICORP_VAULT = "hashicorp/vault:1.15";
+    public static final String HASHICORP_VAULT = "hashicorp/vault:1.15";
```

Comment:
```suggestion
    public static final DockerImageName HASHICORP_VAULT = DockerImageName.parse("hashicorp/vault:1.15");
```

Reply:
Please make io.kroxylicious.kms.provider.hashicorp.vault.TestVault#HASHICORP_VAULT refer to the same symbol too.

Reply:
Mm it is not a trivial change. If I try to get the value from class `VaultTestKmsFacade` it generates a circular dependency between both modules.

Reply:
Fair enough.. We can sort that out later